.circleci/base_config.yml

```diff
@@ -368,6 +379,58 @@ jobs:
       - store_test_results:
           path: /root/project/cppcheck-junit.xml
 
+  build-build-docker-container:
+    docker:
+      - image: cimg/base:current
+    parameters:
+      arch:
+        type: string
+        default: ""
+      resource-class:
+        type: string
+    resource_class: << parameters.resource-class >>
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+          
+      - get-docker-build-scripts
+      - run:
+          name: Build Docker Image
+          command: |
+            case "<< parameters.arch >>" in
+              amd64)
+                echo "Building amd64 image"
+                /bin/sh build-image_x86-64.sh
+              ;;
+              arm64)
+                echo "Building arm64 image"
+                /bin/sh build-image_arm64.sh
+              ;;
+              *)
+                echo "Unknown architecture '$1'"
+                exit 15
+                ;;
+            esac
+      - run:
+          name: Push Docker Image
+          command: |
+            exit 0
+
+
+  build-build-docker-manifest:
+    docker:
+      - image: cimg
+    steps:
+      - get-docker-build-scripts
+      - run:
+          name: Build manifest
+          command: |
+            IMAGE_TAG=${CIRCLE_SHA1::8} ./build-manifest.sh 
```

Comment:
IMAGE_TAG for `build-manifest.sh ` is expected to be `${OS_VERSION}-gcc${GCC_VERSION}-openssl${OPENSSL_VERSION}-${CIRCLE_SHA1::8}`.